observability/logging/logging-6.0/log6x-otel-data-model.adoc

```diff
@@ -0,0 +1,378 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-otel-data-model"]
+= OTEL data model
```

Comment:
While OTEL is a widely used acronym, we should stick using `OpenTelemetry` as the qualifier everywhere here because it is easier to identify for novice users searching through our docs as well as the official name of the standard we are aiming to follow.